src/components/Dropdown/styles/_styles.scss

```diff
@@ -0,0 +1,35 @@
+.dropdown__btn {
+	 @include cx-btn--size(34px);
+	 margin-bottom: 0px;
+	 position: relative;
+	 background-color: $gb-color-component-dark;
+	 color: black;
+	 border: none;
+	 	&:hover{
+	 		cursor: pointer;
+	 	}	  
+}
+.dropdown-content {
+	display: block;
+	margin-top: 0px;
+	width: 200px;
+	background-color: $gb-color-component-dark;
+	color: black;
```

Comment:
Where from color: black is? It's not in the sketches - we use pre-defined colors in `graphen` - please see default text color.